src/modules/common/renderHtml.js

```diff
@@ -10,6 +10,11 @@ const renderJS = (files: PageResourceCacheStateType) => `
   var hrefs = document.querySelectorAll('[href]')
   var srcs = document.querySelectorAll('[src]')
   var files = ${JSON.stringify(files)}
+    
+    if (!files) {
+      console.warn('Files for WebView undefined! Not replacing urls...' )
```

Comment:
Is it just a problem if undefined? Or also if files are empty? If the latter, maybe mention it.

Reply:
It's not a problem if the files are empty. But in this case the remaining part of the function has no effect, so we might just as well `return` in this case.
Do you want me to change the error message?

Reply:
If it's empty, it's not necessarily a faulty state, is it?

Reply:
It's not. But in this case the remaining lines of the function have 0 effect.

Reply:
Right, but no need to do anything special here

Reply:
So you suggest to check for `files === undefined` instead, right?

Reply:
No, I suggest not to change it :D `!files` is only true, if `files` is `undefined` or `null`. If `files` is an empty object, it evaluates to `true`.

Reply:
Ok nvm. I think we both mean the same thing then :D

Reply:
Yeah, me too :D